docs/_rpc/ns-eth.md

```diff
@@ -8,6 +8,28 @@ Geth provides several extensions to the standard "eth" JSON-RPC namespace.
 * TOC
 {:toc}
 
+### eth_createAccessList
```

Comment:
This should not be the first topic on this page, better move it down

Reply:
Please see latest commit.